e-store/e-store/Generated/Assets.swift

```diff
@@ -0,0 +1,189 @@
+// swiftlint:disable all
+// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
+
+#if os(macOS)
```

Comment:
plz create folder called Resources then move all files of assets , localization , Strings and Assets